handbook.md

```diff
@@ -0,0 +1,233 @@
+
+Submit changes incrementally. If you are submitting a pull request that will break compatibility with older APIs, spend the effort to make it compatible first. If you think your feature is not ready for prime time yet, it's OK to submit small changes and hide the feature using feature flags.
+
+Feel free to bring the attention of reviewers by calling them using '@' on GitHub.
+
+### Reviewers
+
+#### Style
+
+Be open to change, but respect the borders between core and plugins. Refer to [What exactly is core?](handbook.html#Whatexactlyiscore?) for that.
+
+Be strict about tests. The project has rewuired tests for most changes to the Ruby code base, however we have not been historically as strict with UI changes. Ask for integration tests for any change that could break in future versions without tests. Testing is a powerful weapon to ensure regressions don't happen.
+
+Be mindful of the use of the word 'you'. It can be easily misunderstood as you reviewing the 'person' not the 'code'. Try not to personalize any references to code. I.e: 'you changed that in line 42' vs 'that change in line 42'.
+
+Back up your reviews by facts when they are not obvious or when contributors might not know why they should make some change. Adopt a "teaching" approach, but again, back up your 'teachings' with links to other sources.
+
+Use '@' to raise the attention of the contributor about any issue. If an user is not 'watching' the project on GitHub or is not subscribed to a particular issue, GitHub will not show notifications nor send emails to that user.
+
+If there is conflict, point to this handbook for reference.
+
+#### Checklist
+http://projects.theforeman.org/projects/foreman/wiki/Reviewing_patches
+
+#### Labels
+
+Labels help reviewers understand what is the status of a pull request. Thanks to them, reviewers can make sure no pull request remains unreviewed and when they get updated, they see it.
+
+* PR processor will set the labels `Not reviewed` and `Needs testing` when a pull request has just been submitted.
+* After you have reviewed the pull request, mark it as 'Waiting on contributor' if it needs changes or you're expecting answers.
+* When the pull request branch gets updated, PR processor will automatically set the labels 'Needs re-review' and 'Needs testing'.
+* Repeat the process until the code is ready to be merged.
+* If there is a problem such as a serious disagreement between people about a change, or there's a dependency on other project that needs to be solved, set it as 'Reached an impasse'.
+
+## Becoming a maintainer
+
+As a maintainer, your responsibility is to keep a project running, taking care of certain parts of it. Examples of such parts are:
+* A plugin, by taking care of documentation, code reviews, and packages to foreman-packaging
```

Comment:
Missing line break between the intro and the list, it's preventing correct formatting.